types/lodash/index.d.ts

```diff
@@ -7221,296 +7257,879 @@ declare namespace _ {
+
+    interface LoDashExplicitObjectWrapperBase<T, TObject extends T | null | undefined, TWrapper> {
+        /**
+         * @see _.flatMapDeep
+         */
+        flatMapDeep<T, TResult>(
+            iteratee: ListIterator<T, Many<TResult>>|DictionaryIterator<T, Many<TResult>>|NumericDictionaryIterator<T, Many<TResult>>
+        ): LoDashExplicitArrayWrapper<TResult>;
+
+        /**
+         * @see _.flatMapDeep
+         */
+        flatMapDeep<TResult>(
+            iteratee: ObjectIterator<any, Many<TResult>>|string
+        ): LoDashExplicitArrayWrapper<TResult>;
+
         /**
-         * @see _.findLast
+         * @see _.flatMapDeep
          */
-        findLast(
-            predicate?: ListIterator<T, boolean>,
-            fromIndex?: number
-        ): any;
+        flatMapDeep<TWhere extends Object>(
+            iteratee: TWhere
+        ): LoDashExplicitArrayWrapper<boolean>;
 
         /**
-         * @see _.findLast
+         * @see _.flatMapDeep
          */
-        findLast(
-            predicate?: string,
-            fromIndex?: number
-        ): any;
+        flatMapDeep(
+            iteratee: [string, any]
+        ): LoDashExplicitArrayWrapper<boolean>;
 
         /**
-         * @see _.findLast
+         * @see _.flatMapDeep
          */
-        findLast(
-            predicate?: PartialObject<T>,
-            fromIndex?: number
-        ): any;
+        flatMapDeep<TResult>(): LoDashExplicitArrayWrapper<TResult>;
     }
 
-    //_.flatMap
+    //_.flatMapDepth
     interface LoDashStatic {
         /**
-         * Creates an array of flattened values by running each element in collection through iteratee
-         * and concating its result to the other mapped values. The iteratee is invoked with three arguments:
-         * (value, index|key, collection).
+         * This method is like `_.flatMap` except that it recursively flattens the
+         * mapped results up to `depth` times.
          *
-         * @param collection The collection to iterate over.
-         * @param iteratee The function invoked per iteration.
-         * @return Returns the new flattened array.
+         * @static
+         * @memberOf _
+         * @since 4.7.0
+         * @category Collection
+         * @param {Array|Object} collection The collection to iterate over.
+         * @param {Function} [iteratee=_.identity] The function invoked per iteration.
+         * @param {number} [depth=1] The maximum recursion depth.
+         * @returns {Array} Returns the new flattened array.
+         * @example
+         *
+         * function duplicate(n) {
+         *   return [[[n, n]]];
+         * }
+         *
+         * _.flatMapDepth([1, 2], duplicate, 2);
+         * // => [[1, 1], [2, 2]]
          */
-        flatMap<T, TResult>(
+        flatMapDepth<T, TResult>(
             collection: List<T> | null | undefined,
-            iteratee?: ListIterator<T, Many<TResult>>
+            iteratee?: ListIterator<T, Many<TResult>>,
```

Comment:
Sorry, If i replace `Many` with `ListOfRecursiveArraysOrValues` within updated functions, tests can failed. I do not make it to adjust the tests to work with `ListOfRecursiveArraysOrValues`. How about you issue another PR to improve this?